.changeset/thin-jeans-wink.md

```diff
@@ -0,0 +1,10 @@
+---
+"@spectrum-css/coachindicator": patch
+"@spectrum-css/dialog": patch
+"@spectrum-css/menu": patch
+"@spectrum-css/progressbar": patch
+"@spectrum-css/progresscircle": patch
+"@spectrum-css/swatch": patch
```

Comment:
The diff script isn't picking this one up as a change. Just making sure that it should still get a bump.

Reply:
Oh I saw Swatch diff locally but I realize now that was a change from a different branch